tests/fixtures/feed_fixtures.py

```diff
@@ -7,4 +7,8 @@ def feed(request):
 
 @pytest.fixture()
 def real_feed(request):
-    return feed_factories.TechCrunchFeedFactory()
\ No newline at end of file
+    return feed_factories.TechCrunchFeedFactory()
+
+@pytest.fixture()
+def feedWithItems(request):
+    return feed_factories.FeedFactory(items=[feed_factories.FeedFactory()])
```

Comment:
How does this call actually give items to a feed factory? I see the function in FeedFactory(), but I'm unsure as to where items are actually being made/added.